src/server/worker.cc

```diff
@@ -75,7 +75,9 @@ Worker::Worker(Server *svr, Config *config) : svr_(svr), base_(event_base_new())
 }
 
 Worker::~Worker() {
-  std::list<Redis::Connection *> conns;
```

Comment:
It will be better to use the list if we need to remove the connection with the pointer?

Reply:
This is a temporary local list, and we only use `emplace_back`. I think there is no reason we remove something from it.

Reply:
And, even if we will do some `remove` from it (although I think it is not possible since we just copy some pointer from other to construct it, and later it is destroyed), we can change its type later.